Emby.Server.Implementations/Cryptography/CryptographyProvider.cs

```diff
@@ -1,40 +1,149 @@
-using System;
```

Comment:
I think this is a \n to \r\n change, please change it back.. I think you can do some settings in your editor or even in git to prevent this